storm-buildtools/storm_checkstyle.xml

```diff
@@ -46,6 +46,9 @@
  -->
 
 <module name = "Checker">
+    <!-- Filter out Checkstyle warnings that have been suppressed with the @SuppressWarnings annotation -->
```

Comment:
I just add allowing SuppressWarnings annotation, since the test code contains long string constant. I didn't recognize that maven checkstyle doesn't check test codes, but it might be useful for specific situations anyway.